examples/205-automl.ipynb

```diff
@@ -3,7 +3,11 @@
   {
```

Comment:
<p>It could be better:</p><p>This custom dict could be passed into <code>Tune</code> class...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Done